Src/AutoFixture.xUnit.net.UnitTest/CompositeDataAttributeSufficientDataTest.cs

```diff
@@ -63,17 +63,6 @@ public IEnumerator<object[]> GetEnumerator()
             yield return CreateTestCase(
                 data: new[]
                     {
-                        new FakeDataAttribute(this.method, this.parameterTypes, new[] { new object[] { 1, 2, 3, 4 } })

```

Comment:
See [this](https://github.com/AutoFixture/AutoFixture/pull/356/files#r25605375) comment. – Would it make sense to run the test once, and feed it with `1`, `2`, `3`, while throwing `4` out?